pandas/_libs/tslibs/offsets.pyx

```diff
@@ -2738,13 +2738,28 @@ cdef class BYearBegin(YearOffset):
     _day_opt = "business_start"
 
 
-cdef class YearEnd(YearOffset):
+cdef class _YearEnd(YearOffset):
```

Comment:
Can you add a comment that this class exists due to https://github.com/cython/cython/issues/3873

Reply:
Addressed in https://github.com/pandas-dev/pandas/pull/61605/commits/0cb108efc22597128c80257b4c5ca47cd5bc7134